### PR TITLE
Bumps in ionic : ci_matching_branch/bump_ionic_gz-ionic

### PR DESCRIPTION
**Pull request by azeey**
See https://github.com/gazebo-tooling/release-tools/issues/1027

### CODE REVIEW

File: gazebodistro/collection-ionic.yaml
Comment (iche033) [inline gazebodistro/collection-ionic.yaml:1]: should we keep the collection yaml here or just have an empty release notes file?

Comment (azeey) [reply gazebodistro/collection-ionic.yaml:1]: Yup, I'll do that.

File: .github/ci/packages.apt [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+libgz-sensors9-dev
+libgz-sim9-dev
 libgz-tools2-dev
-libgz-transport13-dev
-libgz-utils2-cli-dev
-libgz-utils2-dev
+libgz-transport14-dev
+libgz-utils3-cli-dev
+libgz-utils3-dev
 libsdformat13-dev
```
Comment (iche033) [inline .github/ci/packages.apt]:
```suggestion
libsdformat14-dev
```

File: .github/ci/packages.apt [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+gz-transport14-cli
+libgz-cmake4-dev
+libgz-common6-dev
+libgz-fuel-tools10-dev
+libgz-gui9-dev
+libgz-launch8-dev
+libgz-math8-dev
+libgz-math8-eigen3-dev
+libgz-msgs11-dev
 libgz-physics6-dev
```
Comment (iche033) [inline .github/ci/packages.apt]:
```suggestion
libgz-physics8-dev
```

File: .github/ci/packages.apt [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-libgz-utils2-dev
+libgz-transport14-dev
+libgz-utils3-cli-dev
+libgz-utils3-dev
 libsdformat13-dev
-python3-gz-math7
-python3-gz-sim8
+python3-gz-math8
+python3-gz-sim9
 python3-sdformat13
```
Comment (iche033) [inline .github/ci/packages.apt]:
```suggestion
python3-sdformat14
```

**Review by iche033 — APPROVED** [review]

**Pull request merged**